### ISSUE TITLE
[Bug]: config doesn't effect the game

### ISSUE THREAD

**Issue by Idkijustwannatellu** [Silentine/GrimoireOfGaia#227]
### General Info

- [x] I am running a modpack
- [ ] I can reproduce this issue consistently in single-player
- [ ] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [ ] I am using the latest version of the mod

### Forge version

1.12.1

### Mod version

4.0.0 - 1.12.1

### Java version

1.12.1

### Issue Description

changed the spawn rate to some mobs to 0 but they still spawn

**Comment by Mrbysco** [comment]
What version of minecraft are you playing? Because 4.0.0 does not exist on 1.12

**Comment by Mrbysco** [comment]
Also did you restart your game? as it won't update live

**Comment by Idkijustwannatellu** [comment]
> Also did you restart your game? as it won't update live

yes I did restart it I didn't update it live, and sorry idk what verison minecraft its just 1.12.1, everythings put together correctly and working, it's just changing the config doesn't seem to do anything.
